app/src/main/java/com/jerboa/feat/BlurNsfwTypes.kt

```diff
@@ -0,0 +1,27 @@
+package com.jerboa.feat
+
+import android.os.Parcelable
+import androidx.annotation.StringRes
+import com.jerboa.R
+import com.jerboa.datatypes.types.PostView
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+enum class BlurNsfwTypes(@StringRes val resId: Int) : Parcelable {
+    Nothing(R.string.app_settings_nothing),
+    All(R.string.account_settings_all),
+    OnlyFromNsfwCommunity(R.string.app_settings_blur_only_from_nsfw_community),
+    OnlyFromSfwCommunity(R.string.app_settings_blur_only_from_sfw_community),
```

Comment:
Not sure I understand this one. Only blur items from safe communities? I don't think that needs to be an option.

Reply:
That one means only blur NSFW posts from SFW communities.